src/other/acr/demo-wcag21/labels.json

```diff
@@ -0,0 +1,430 @@
+{
+  "labels": {
+    "@context": {
+      "ATAG:": "https://www.w3.org/TR/ATAG20/#"
+    },
+    "acr:ConformanceStandard": {
+      "en": {
+        "https://www.w3.org/TR/WCAG21": "WCAG 2.1",
+        "https://www.w3.org/TR/WCAG21/#new-features-in-wcag-2-1": "Delta of WCAG 2.1",
+        "https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/": "EN 301 549 v3.2.1"
+      },
+      "fr": {
+        "https://www.w3.org/TR/WCAG21": "WCAG 2.1",
+        "https://www.w3.org/TR/WCAG21/#new-features-in-wcag-2-1": "Delta of WCAG 2.1",
+        "https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/": "EN 301 549 v3.2.1"
+      }
+    },
+    "acr:ConformanceOption": {
+      "en": {
+        "act:standard/wcag21/levelAA": "Level AA",
+        "act:standard/en301_549/ictTwoVoiceComm": "ICT with two-way voice communication",
+        "act:standard/en301_549/ictVideo": "ICT with video capabilities",
+        "act:standard/en301_549/hardware": "Hardware",
+        "act:standard/en301_549/web": "Web",
+        "act:standard/en301_549/nonWeb": "Non-web documents",
+        "act:standard/en301_549/software": "Software",
+        "act:standard/en301_549/docAndSupportServices": "Documentation and support services",
+        "act:standard/en301_549/ictRelayOrEmergency": "ICT providing relay or emergency service access"
+      },
+      "fr": {
+        "act:standard/wcag21/levelAA": "Niveau AA",
+        "act:standard/en301_549/ictTwoVoiceComm": "ICT with two-way voice communication",
+        "act:standard/en301_549/ictVideo": "ICT with video capabilities",
+        "act:standard/en301_549/hardware": "Hardware",
+        "act:standard/en301_549/web": "Web",
+        "act:standard/en301_549/nonWeb": "Non-web documents",
+        "act:standard/en301_549/software": "Software",
+        "act:standard/en301_549/docAndSupportServices": "Documentation and support services",
+        "act:standard/en301_549/ictRelayOrEmergency": "ICT providing relay or emergency service access"
+      }
```

Comment:
Match the term defined at :

* https://wet-boew.github.io/act-rules/standard_profiles/wcag
* https://wet-boew.github.io/act-rules/standard_profiles/en301549

Reply:
Les termes ont été ajusté et documenté. Par example ici: https://github.com/wet-boew/act-rules/blob/main/src/standard/profiles/en301549.md